### ISSUE TITLE
[Bug]: Cannot find color scheme 'lunar' (regression?)

### ISSUE THREAD

**Issue by etienne-k** [ChristianChiarulli/LunarVim#792]
### Problem description

I suspect the bug described in #554 and probably corrected by 9532d33d50017ec213ed17ec95cef87b9b6c9b47 was reintroduced by ec814f60c55a8d9d4695bdde1240289b2743f01a.

In a clean environment, this leads to the startup error described in #554.

If I'm not wrong, ec814f60c55a8d9d4695bdde1240289b2743f01a moves the theme loading back to a point before packer is invoked.

Neovim version:
```
NVIM v0.5.0
Build type: Release
LuaJIT 2.0.5
Compilation: /usr/bin/cc -D_FORTIFY_SOURCE=2 -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -DNVIM_TS_HAS_SET_MATCH_LIMIT -O2 -DNDEBUG -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=always -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/build/neovim/src/neovim-0.5.0/build/config -I/build/neovim/src/neovim-0.5.0/src -I/usr/include -I/build/neovim/src/neovim-0.5.0/build/src/nvim/auto -I/build/neovim/src/neovim-0.5.0/build/include
```

Running on Arch with 5.12.11-zen1-1-zen.

### LunarVim version

master

### Neovim version (>= 0.5)

0.5.0

### Relevant log output

```shell
Error while executing "/usr/share/nvim/runtime/syntax/synload.vim":
Line   19:
Cannot find color scheme 'lunar'
```

### I have

- [X] Read the readme
- [X] Read the wiki
- [X] Searched for similar issues

**Comment by rebuilt** [comment]
Thanks! That setting keeps getting moved around.

**Comment by rebuilt** [comment]
In the meantime, the the short-term fix is

```
git clone https://github.com/ChristianChiarulli/nvcode-color-schemes.vim ~/.local/share/nvim/site/pack/packer/opt/nvcode-color-schemes.vim
```

**Comment by rebuilt** [comment]
Fixed in master https://github.com/ChristianChiarulli/LunarVim/pull/798

**Comment by rebuilt** [comment]
Fixed in master #798
